apps/openmw/mwmechanics/actors.cpp

```diff
@@ -374,9 +374,13 @@ namespace MWMechanics
             if (LOS)
             {
                 MWBase::Environment::get().getMechanicsManager()->startCombat(actor1, actor2);
-                if (!againstPlayer) // start combat between each other
+                if (!againstPlayer) // start combat between each other if engaging actor is not one of the player followers
```

Comment:
More specifically this is done because `actorAttacked()` checks for `!seq.isInCombat(attacker)` before calling `commitCrime()`.

Which will of course fail when startcombat is issued in here also for actor2.